src/toil/test/provisioners/clusterScalerTest.py

```diff
@@ -308,23 +307,12 @@ def getNumberOfJobsIssued(self):
         # AbstractScalableBatchSystem functionality
 
         def getNodes(self):
-            return {address: NodeInfo(coresTotal=0, coresUsed=0, requestedCores=1,
-                                      memoryTotal=0, memoryUsed=0, requestedMemory=1,
-                                      workers=1 if w.busyEvent.is_set() else 0)
-                    for address, w in enumerate(self.workers)}
-
-        # AbstractProvisioner functionality
-
-        def setNodeCount(self, numNodes):
```

Comment:
The setNodeCount method got moved away from the provisioner a while ago, but I guess no one removed the dead code in this test. I only noticed it was dead when changing the way `_removeNodes` was called.